### DISCUSSION TITLE
Was soll alles in die Präsentation?

### DISCUSSION THREAD

**Discussion by Marc738** [SE-TINF22B2/G3-ApeRepublic#84] [General]
Brainstorm Threat

**Comment by Marc738** [comment]
Testprojekte (Angular, Spring Boot, Deno)

**Comment by Marc738** [comment]
Vorstellung Spezifizierung von UserStory zu Use Case

**Comment by elnaz-gharoon** [comment]
Vorstellung Project Ziele (Visions ) -> Am Anfang

**Comment by elnaz-gharoon** [comment]
Project Managment und lessons learned

**Comment by codajoda** [comment]
Mockups
